examples/lm1b/README.md

```diff
@@ -66,13 +66,23 @@ gcloud compute ssh $USER-user-vm-0001 -- -L 2222:localhost:8888
 
 Be sure to install the latest `jax` and `jaxlib` packages alongside the other
 requirements above. e.g. as of April 2020 the following package versions were
-used successfully: `pip install -U pip pip install -U setuptools wheel pip
-install -U pip jax jaxlib sentencepiece \ tensorflow==2.2.0rc3
-tensorflow-datasets==3.0.0 \ tensorflow-text==2.2.0rc2 tensorboard git clone
-https://github.com/google/flax pip install -e flax`
+used successfully:
```

Comment:
It would be great to add a link to tfhub.dev from your successful run to the README, similar to [imagenet/README.md](https://github.com/google/flax/blob/master/examples/imagenet/README.md)

You can use a command similar to below:

```
mkdir -p /tmp/lm1b/$IDENT
gsutil cp $WORKDIR/*.tfevents.* /tmp/lm1b/$IDENT
tensorboard dev upload --one_shot --logdir /tmp/lm1b/ --name "Flax lm1b example" --description "https://github.com/google/flax/pull/1329"
```

Reply:
I actually didn't complete the run, I was testing temperature sampling and just made sure the example ran again. If I end up running it I'll put it here!